### PR TITLE
Refactor options

### PR DESCRIPTION
**Pull request by notdryft**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -20,7 +20,7 @@ repositories {
 }
 
 dependencies {
-  implementation "io.gatling:gatling-enterprise-plugin-commons:1.9.6"
+  implementation "io.gatling:gatling-enterprise-plugin-commons:1.9.7-SNAPSHOT"
```
Comment (notdryft) [inline build.gradle:23]: Wait for release before merging

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Set up JDK21
+      - name: Set up JDK11
```
Comment (slandelle) [inline .github/workflows/publish.yml:14]: Why?

**Review by slandelle — APPROVED** [review]

**Comment by guilgaly** [comment]
See gatling/gatling-enterprise-plugin-commons#149 for teh corresponding changes in the enterprise-commons library

File: src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy [hunk 3]
```diff
@@ -220,12 +209,12 @@ class GatlingPluginExtension {
         }
 
         private PluginConfiguration getPluginConfiguration(Logger logger, boolean forceBatchMode) {
-            final apiToken = getApiToken()
+            final apiToken = ConfigurationConstants.ApiToken.value()
             if (!apiToken) {
                 throw new InvalidUserDataException("""
                     |An API token is required to call the Gatling Enterprise server.
                     |See https://gatling.io/docs/enterprise/cloud/reference/admin/api_tokens/ and create a token wil the role 'Configure'.
```
Comment (guilgaly) [inline src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy:216]: Dead doc link, we can fix it too while we're at it. 😉

**Review by guilgaly — APPROVED** [review]

**Pull request merged**